dali/base/dadfs.cpp

```diff
@@ -4704,6 +4705,20 @@ protected: friend class CDistributedFilePart;
         else
             return false;
     }
+    virtual double getCost(const char * cluster) override
+    {
+        VStringBuffer path("storage/planes[@name='%s']", cluster);
```

Comment:
What happens if cluster is null/empty?  Should it iterate through all the clusters the file lives on?  (I think cluster can be null on the call to dfuInfo).

Reply:
> ghalliday: What happens if cluster is null/empty? Should it iterate through all the clusters the file lives on? (I think cluster can be null on the call to dfuInfo).

I think this is still an issue.
DFUInfo will normally come with a particular cluster specified, but the file is still on 1 or more clusters.

As Gavin says, if cluster is an null/empty, I think needs to iterate through the clusters the file is on, and total up the cost.

Reply:
@shamser - can you comment on this one?

Reply:
@jakesmith I've implemented this (re iterating through all clusters) in the latest commit.